### DISCUSSION TITLE
Detect calls to routes that do not exist

### DISCUSSION THREAD

**Discussion by gms8994** [larastan/larastan#1603] [Ideas]
It would be really nice if it were possible to have some code that points to a route that doesn't exist; for example, a call to `route('this.thing.doesnt.exist')` and triggering an message.

**Comment by szepeviktor** [comment]
This command check the controller of each route. This is not static analysis.
https://github.com/szepeviktor/debian-server-tools/blob/master/webserver/laravel/Commands/RouteCheckCommand.php
I understand you want something else.

**Reply by gms8994** [reply]
I'm not wanting to check the controller, I want to check to see if the called route exists as a defined route. Similar to seeing if a class exists.
